src/js/views/messageView.js

```diff
@@ -26,11 +29,23 @@ module.exports = Marionette.ItemView.extend({
                 }
             },
             '@ui.message': {
-                observe: 'text',
-                update: function($el, text) {
-                    var escapedText = $('<div/>').text(text).html().replace(/\n/g, '<br />');
+                observe: ['text', 'actions'],
+                update: function($el, values) {
+                    if (values[0].trim().length > 0) {
```

Comment:
No, if no text was provided (just an action), we don't care about rendering anything. The `span` will remain empty.